lib/reek/smells/uncommunicative_variable_name.rb

```diff
@@ -61,10 +62,15 @@ def sniff(ctx)
         end
       end
 
-      def bad_name?(name, _ctx)
-        var = name.to_s.gsub(/^[@\*\&]*/, '')
-        return false if accept_names.include?(var)
-        reject_names.find { |patt| patt =~ var }
+      private
+
+      def uncommunicative_variable_name?(name)
+        !acceptable_name?(name)
+      end
+
+      def acceptable_name?(name)
```

Comment:
I know this seems unnecessary. The method body from here could go into `uncommunicative_variable_name` directly. But I find positive predicate method names easier to read and it's easier to build up a mental image of them.